components/atom/button/src/index.scss

```diff
@@ -93,11 +93,6 @@ $tt-atom-button: none !default;
 // Set all color properties for text and icons
 @mixin button-foreground-color($color) {
   color: $color;
-
-  & svg path {
```

Comment:
This was an overkill and unnecessary as the color of the icon is the same as the text so the css for this could be used once instead creating it for every option.